tests/test_core.py

```diff
@@ -1475,70 +1475,10 @@ def test_bigarray(self):
   def test_mod_globalstruct(self):
     self.do_run_in_out_file_test('tests', 'core', 'test_mod_globalstruct')
 
-  @no_wasm_backend()
+  @no_wasm_backend('specific alignment semantics may be asmjs-specific?')
```

Comment:
Oh, heh, I missed that, sorry.

(If that's the issue here, then you can fix the test by adding another correct output for it, in wasm-backend mode.)